typings/index.d.ts

```diff
@@ -82,6 +82,15 @@ declare module 'discord.js' {
     public split(): APIMessage[];
   }
 
+  class TypingData {
+    public readonly client: Client;
+    public since: Date;
+    public lastTimestamp: Date;
+    private _timeout: NodeJS.Timeout;
+    public resetTimeout(_timeout: NodeJS.Timeout): void;
+    public readonly elapsedTime: number;
```

Comment:
these properties could be organised alphabetically, functions declared after all the properties and private properties at the top however

also the constructor and its parameters defined at the top

Reply:
Typings have been updated.